README.md

```diff
@@ -14,6 +14,7 @@ Check out __[Running React Native everywhere](https://mmazzarolo.com/blog/2021-0
 
 <p align="center" margin-bottom="0">
   <img width="820" height="auto" src="./.github/images/all-screenshot.png">
+  <img width="820" height="auto" src="./.github/images/tv.png">
```

Comment:
Can we remove this? We can keep the image in `images/` but I don't wanna blow up the README.md 👍

Reply:
(I'll take care of updating the README adding references to the new platforms + your blogpost after merging this PR 👍 )

Reply:
You got it 👍🏽